.github/workflows/release-manual.yml

```diff
@@ -0,0 +1,28 @@
+name: release-manual
+on:
+  release:
+    types:
+      - published
+jobs:
+  release:
```

Comment:
Can you duplicate the job `release`, rename it `dry-run`, and add the following to the job `release`:
```
release:
   needs: dry-run
```

Reply:
done
For testing, we could just create a github (pre)release with a code where the versions in the pubspec.yaml files have already been published.
Nothing could go wrong, since pub.dev does not allow to update a release.

Reply:
Yep!